File: backend/ms-reservas/ms-reservas/src/main/java/com/example/ms_reservas/model/Mesa.java
package com.example.ms_reservas.model;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "mesa")
@Data
public class Mesa {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_mesa")
    private Long idMesa;

    @Column(name = "numero", nullable = false)
    private Integer numero;

    @Column(name = "capacidad", nullable = false)
    private Integer capacidad;

    @Column(name = "disponible", nullable = false)
    private Boolean disponible;
}

File: app/src/main/java/com/example/sakura_fusion/data/mesa/Mesa.kt
package com.example.sakura_fusion.data.mesa

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity(tableName = "mesa")
data class Mesa(
    @PrimaryKey
    @ColumnInfo(name = "id_mesa")
    @SerializedName("id_mesa")
    val idMesa: Int,
    val numero: Int,
    val capacidad: Int,
    val disponible: Boolean
)
